packages/atomic/tailwind.config.js

```diff
@@ -55,5 +57,16 @@ module.exports = {
       backgroundColor: ['group-focus'],
     },
   },
-  plugins: [],
+  plugins: [
+    plugin(function ({addUtilities, theme}) {
+      addUtilities(
+        {
+          '.outline-color-primary-light': {
```

Comment:
Only one color for the outline is necessary and should be reused on elements displaying an outline.

Reply:
If there is only one, could the name be shortened to `.outline-color`?